### PR TITLE
ch.are.belastung-personenverkehr

### PR DESCRIPTION
**Pull request by ltclm**
new symbology and activate tooltip for:
* ch.are.belastung-personenverkehr-strasse
* ch.are.belastung-personenverkehr-bahn

[Testlink](https://mf-chsdi3.dev.bgdi.ch/feature_are_belastung_personenverkehr/shorten/88845b6578)

https://jira.swisstopo.ch/browse/BGDIDI_KB-2500

### CODE REVIEW

**Comment by ltclm** [comment]
once merged we can rebase the other pending pr and fix jenkins tests in there.

File: chsdi/templates/htmlpopup/personenverkehr_strasse.mako [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<%def name="table_body(c, lang)">
+<% c['stable_id'] = False %>
+    <tr><td class="cell-left">${_('ch.are.belastung-personenverkehr-strasse.nr')}</td>          <td>${c['attributes']['nr']}</td></tr>
+    <tr><td class="cell-left">${_('ch.are.belastung-personenverkehr-strasse.dwv_fzg')}</td>     <td>${c['attributes']['dwv_fzg']}</td></tr>
+</%def>
+
+<%def name="extended_info(c, lang)">
+<table class="table-with-border kernkraftwerke-extended">
+ <tr><th class="cell-left">${_('ch.are.belastung-personenverkehr-strasse.nr')}</th><td>${c['attributes']['nr'] or '-'}</td></tr>
```
Comment (rebert) [inline chsdi/templates/htmlpopup/personenverkehr_strasse.mako:11]: I would use <th /> in the first row of a table and not in the first column

File: chsdi/models/vector/are.py [hunk 2]
```diff
@@ -173,3 +173,54 @@ class ZweitwohnungsAnteil(Base, Vector):
...
+    msp_lw = Column('msp_lw', Integer)
+    msp_lz = Column('msp_lz', Integer)
+    asp_fzg = Column('asp_fzg', Integer)
+    asp_pw = Column('asp_pw', Integer)
+    asp_li = Column('asp_li', Integer)
+    asp_lw = Column('asp_lw', Integer)
+    asp_lz = Column('asp_lz', Integer)
+    the_geom = Column(Geometry2D)
+
+register('ch.are.belastung-personenverkehr-strasse', BelastungPersonenverkehrStrasse)
```
Comment (rebert) [inline chsdi/models/vector/are.py:209]: BelastungPersonenverkehrStrasse.__bodId__ could be used instead

Comment (ltclm) [reply chsdi/models/vector/are.py:209]: ok thank you!

File: chsdi/templates/htmlpopup/personenverkehr_strasse.mako (hunk 1 above)
Comment (ltclm) [reply chsdi/templates/htmlpopup/personenverkehr_strasse.mako:11]: sorry, i dont understand:

**Comment by ltclm** [comment]
important:
THIS PR is using a custom bod. Do not merge production.ini.in into data.
After merging reset the main bod with the following command:
```sql
UPDATE re3.layers_js SET tooltip = TRUE WHERE fk_id_dataset in ('ch.are.belastung-personenverkehr-bahn', 'ch.are.belastung-personenverkehr-strasse');
```

**Pull request merged**